caom2pipe/manage_composable.py

```diff
@@ -469,10 +434,10 @@ def capture(self):
             create_dir(self.observable_dir)
             now = datetime.utcnow().timestamp()
             for service in self.history.keys():
-                fqn = '{}/{}.{}.yml'.format(self.observable_dir, now, service)
+                fqn = f'{self.observable_dir}/{now}.{service}.yml'
```

Comment:
I tend to use `os.path.join(dir, filename)` just to make the code portable. I know it's not the case here but it's a good practice.

Reply:
Replaced with os.path.join. Tried to catch all the instances in the file.